src/amqp/queue.js

```diff
@@ -11,6 +11,9 @@ var topLog = require( "../log" )( "rabbot.topology" );
 var unhandledLog = require( "../log" )( "rabbot.unhandled" );
 var noOp = function() {};
 
+// Disable caching because of memory leak
+postal.configuration.resolverNoCache = true;
```

Comment:
Actually... we don't need to patch rabbot for this; we can set this option in `postal` from our application.